its/pom.xml

```diff
@@ -29,9 +29,7 @@
       <build>
         <plugins>
           <plugin>
-            <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-dependency-plugin</artifactId>
-            <version>2.10</version>
```

Comment:
@vilchik-elena 
groupId is not required for Maven native plugins.
the version of all Maven plugins is inherited from the parent POM org.sonarsource.parent:parent